api.go

```diff
@@ -131,10 +139,12 @@ func (api *API) status(w http.ResponseWriter, req *http.Request) {
 
 	var status = struct {
 		Config      string            `json:"config"`
+		ConfigJson  config.Config     `json:"configJson"`
 		VersionInfo map[string]string `json:"versionInfo"`
 		Uptime      time.Time         `json:"uptime"`
 	}{
 		Config: api.config,
+		ConfigJson: api.configJson,
```

Comment:
Done. I also changed the visible JSON object member, please tell me if you prefer to keep the old one.